2022-03-08 00:47:04,882 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 00:47:04,882 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 00:47:04,882 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 00:47:04,882 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 00:47:04,883 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_004704-1m7r0npe/logs/debug.log
2022-03-08 00:47:04,883 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_004704-1m7r0npe/logs/debug-internal.log
2022-03-08 00:47:04,883 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 00:47:04,883 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 00:47:04,883 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: zk3r1tve
2022-03-08 00:47:04,886 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/zk3r1tve
2022-03-08 00:47:04,887 INFO    MainThread:3014 [jupyter.py:save_history():429] not saving jupyter history
2022-03-08 00:47:04,887 INFO    MainThread:3014 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-08 00:47:04,887 INFO    MainThread:3014 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-03-08 00:47:04,887 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 00:47:04,889 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 00:47:05,692 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:47:05,858 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:47:06,098 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:47:06,201 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 17036
}

2022-03-08 00:47:06,306 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 17036
}

2022-03-08 00:47:06,408 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9303
  total_bytes: 17036
}

2022-03-08 00:47:06,511 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:06,615 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:06,717 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:06,820 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:06,922 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:07,462 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}

2022-03-08 00:47:07,874 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17036
  total_bytes: 17036
}
local_info {
}

2022-03-08 00:47:09,347 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
2022-03-08 00:47:09,358 INFO    MainThread:3014 [wandb_init.py:init():471] starting backend
2022-03-08 00:47:09,358 INFO    MainThread:3014 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-08 00:47:09,362 INFO    MainThread:3014 [backend.py:ensure_launched():219] starting backend process...
2022-03-08 00:47:09,469 INFO    MainThread:3014 [backend.py:ensure_launched():225] started backend process with pid: 4789
2022-03-08 00:47:09,470 INFO    MainThread:3014 [wandb_init.py:init():480] backend started and connected
2022-03-08 00:47:09,500 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-08 00:47:09,500 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-08 00:47:09,500 INFO    MainThread:3014 [wandb_init.py:init():550] updated telemetry
2022-03-08 00:47:09,685 INFO    MainThread:3014 [wandb_init.py:init():581] communicating current version
2022-03-08 00:47:11,551 INFO    MainThread:3014 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-08 00:47:11,553 INFO    MainThread:3014 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-08 00:47:11,765 INFO    MainThread:3014 [wandb_init.py:init():624] starting run threads in backend
2022-03-08 00:47:16,776 INFO    MainThread:3014 [wandb_run.py:_console_start():1827] atexit reg
2022-03-08 00:47:16,777 INFO    MainThread:3014 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-08 00:47:16,777 INFO    MainThread:3014 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-08 00:47:16,778 INFO    MainThread:3014 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-08 00:47:16,778 INFO    MainThread:3014 [wandb_init.py:init():651] run started, returning control to user process
2022-03-08 00:51:53,973 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 00:51:53,973 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 00:51:53,973 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 00:51:53,973 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005153-1j6me8k8/logs/debug.log
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005153-1j6me8k8/logs/debug-internal.log
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7faf020f8828>
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 00:51:53,974 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: 1m7r0npe
2022-03-08 00:51:53,976 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/1m7r0npe
2022-03-08 00:51:53,976 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 00:51:53,977 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 00:51:54,681 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:51:54,879 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:51:55,069 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:51:56,095 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 9305
}

2022-03-08 00:51:56,200 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 18011
}

2022-03-08 00:51:56,304 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 18011
}

2022-03-08 00:51:56,406 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 18011
}

2022-03-08 00:51:56,509 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5674
  total_bytes: 18011
}

2022-03-08 00:51:56,613 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:56,717 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:56,819 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:56,921 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:57,024 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:57,126 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:57,634 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}

2022-03-08 00:51:58,056 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18011
  total_bytes: 18011
}
local_info {
}

2022-03-08 00:51:59,519 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
